docs/public/specifications/README.md

```diff
@@ -1,11 +1,10 @@
-![Zonemaster](/docs/images/zonemaster_logo_2021_color.png)
-==========
+# Specifications
 
-### Purpose of this directory
 
-This directory contains detailed technical specifications for the DNS tests performed by Zonemaster.
+* [Tests](tests/): contains the specifications of the *Test Cases* that the
+  Zonemaster implementation is based on.
+* [Test types](test-types/): contains specification *Test Types*, currently only
+  for undelegated tests.
+* [Test zones](test-zones/): contains specifications of *Test zones* for
```

Comment:
Maybe `*Test Zones*` with the first letter uppercased for consistency.

Reply:
Fixed.